.claude/scripts/remote-playwright.sh

```
@@ -0,0 +1,221 @@
+    app: playwright-server
+  ports:
+    - port: 3000
+      targetPort: 3000
+      protocol: TCP
+EOF
+
+  echo "Waiting for rollout..."
+  oc rollout status deployment/playwright-server -n "$NAMESPACE" --timeout=300s
+
+  # Wait for server to be listening
+  for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)